File: python/nivel 5 simulacion/29.py
"""
Ejercicio 29
Contar palabras en una frase
"""
frase = input("Ingresa una frase: ").strip()

if frase =="":
  print("la frase no contiene palabras, esta vacia")
else:
  palabras = frase.split()
  print("Cantidad de palabras: ",len(palabras))

File: javascript/nivel 5 simulacion/29.js
/*
Ejercicio 29
Contar palabras en una frase
*/
let frase = prompt("ingresa una frase: ");

frase = frase.trim();

if(frase === ""){
  console.log("La frase no contiene palabras");
}else{
  let palabras = frase.split(/\s+/);
  console.log("cantidad de palabras: ", palabras.length);
}